src/suricata.c

```diff
@@ -2818,6 +2818,10 @@ static void SuricataMainLoop(SCInstance *suri)
 {
     while(1) {
         if (sigterm_count || sigint_count) {
+#ifdef HAVE_AF_XDP
+            if (suricata.run_mode == RUNMODE_AFXDP_DEV)
+                RunModeAFXDPRemoveProg();
```

Comment:
This probably just needs a comment explaining why the special magic like "this is needed to work around a kernel-crasher in Linux < X.Y"

Reply:
Ok, will add comment and strerrorline